primitives/ethereum-poa/src/lib.rs

```diff
@@ -117,9 +117,9 @@ pub struct UnsignedTransaction {
 	pub gas: U256,
 	/// Transaction destination address. None if it is contract creation transaction.
 	pub to: Option<Address>,
-	/// Transaction value.
+	/// Value.
```

Comment:
This doesn't really add much value in terms of describing what "value" is. I'd beef up the description a bit